static/src/components/fieldsconfig/index.vue

```diff
@@ -0,0 +1,115 @@
+<template>
+    <div class="stats-config row">
+        <div class="col-xs-12">
+            <div class="form-inline">
+                <div class="form-group" :class="{'has-error': error}">
+                    <input type="text" class="form-control input-sm"
+                        v-model="fieldName">
+                    <select name="field-type" class="form-control input-sm"
+                            v-model="fieldType">
+                        <option v-for="(conf, type, index) in labelTypes.config"
+                                :key="index"
+                                :value="type">{{conf.display}}</option>
+                    </select>
+                    <button class="btn btn-sm btn-primary"
+                        :disabled="!fieldName.length"
```

Comment:
should this be fieldNames ?

Reply:
`fieldName` should be right - however `.lenght` is not needed since it is a string.